apps/dotcom/src/utils/sharing.ts

```diff
@@ -241,7 +241,7 @@ async function getRoomData(
 			// processed it
 			if (!asset) continue
 
-			data[asset.id] = await cloneAssetForShare(asset, uploadFileToAsset)
+			data[asset.id] = await cloneAssetForShare(asset, editor, uploadFileToAsset)
```

Comment:
secret style guide nit: whenever editor is an argument to a function, let's make it the first argument